test-runner/src/CommandInput.js

```diff
@@ -21,12 +31,13 @@ function CommandInput({ setInnerHTML, availableCommands }) {
       ...existingHistory,
       {
         content: "",
-        errors: [],
+        consoleOutputs: [],
```

Comment:
Just like on the backend, get rid of tracking errors independently and only track the console outputs. Within each output we have a type which can be `error` or `log` (or potentially `info` etc).